chapters/07_chapter_6.tex

```diff
@@ -0,0 +1,390 @@
+\chapter{Introduction to micro-architectural attacks}
+
+
+
+Clémentine Maurice, CNRS, IRISA\\
+April 30, 2019 — Ben Gurion University, Israel
+
+\label{c7_cacheattacks}
```

Comment:
The label should come after the \chapter and start with \label{chap:XXX}

Reply:
Done